miasm2/arch/aarch64/sem.py

```diff
@@ -4,6 +4,7 @@
 from miasm2.arch.aarch64.regs import *
 from miasm2.core.sembuilder import SemBuilder
 
+EXCEPT_INT_DIV_BY_ZERO = (1 << 16)
```

Comment:
As these constants are relative to Miasm and not to the architecture, we should have them in a single place (which seems to be, for now, `csts.py`)